recipes/tango-database/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "tango-database" %}
+{% set version = "5.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/tango-controls/TangoDatabase/-/archive/Database-Release-{{ version }}/TangoDatabase-Database-Release-{{ version }}.tar.gz
+  sha256: f1585422800a22c7919261425f28992b865152a3756e0d0bcdddc6f0eaa7fc01
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - libtool
+    - pkg-config
+  host:
+    - cppzmq
+    - cpptango
+    - mysql-devel
+  run:
+    # cpptango set in run_exports
+    - {{ pin_compatible('omniorb', min_pin='x.x', max_pin='x.x') }}
```

Comment:
`omniorb` should probably be a `host` dependency

Reply:
I removed the explicit omniorb run dependency. It's already installed as cpptango dependency.

Reply:
You should add it if you're linking to it (which you are). Everything you link to should be explicitly listed in `host` so the bots know how to perform ABI migrations without building the entire world every time.

Reply:
Good point. Thanks. I added it back.